cvat-cli/src/cvat_cli/_internal/commands_projects.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import textwrap
+
+from cvat_sdk import Client, models
+
+from .command_base import CommandGroup, GenericCommand, GenericDeleteCommand, GenericListCommand
+from .parsers import parse_label_arg
+
+COMMANDS = CommandGroup(description="Perform operations on CVAT projects.")
+
+
+class GenericProjectCommand(GenericCommand):
+    resource_type_str = "project"
+
+    def repo(self, client: Client):
+        return client.projects
+
+
+@COMMANDS.command_class("ls")
+class ProjectList(GenericListCommand, GenericProjectCommand):
+    pass
+
+
+@COMMANDS.command_class("create")
+class ProjectCreate:
```

Comment:
Consider adding support for dataset import as well.

Reply:
Alright, added.